packages/paste-core/primitives/combobox/src/index.tsx

```diff
@@ -2,12 +2,20 @@ import Downshift from '@twilio-paste/dropdown-library';
 
 export {Downshift as ComboboxPrimitive};
 
-export {useCombobox as useComboboxPrimitive} from '@twilio-paste/dropdown-library';
+export {
+  useCombobox as useComboboxPrimitive,
+  useMultipleSelection as useMultiSelectPrimitive,
+} from '@twilio-paste/dropdown-library';
 
 export type {
   UseComboboxInterface as UseComboboxPrimitiveInterface,
   UseComboboxProps as UseComboboxPrimitiveProps,
   UseComboboxState as UseComboboxPrimitiveState,
   UseComboboxReturnValue as UseComboboxPrimitiveReturnValue,
   UseComboboxStateChange as UseComboboxPrimitiveStateChange,
+  UseMultipleSelectionInterface as UseMultiSelectPrimitiveInterface,
+  UseMultipleSelectionProps as UseMultiSelectPrimitiveProps,
+  UseMultipleSelectionState as UseMultiSelectState,
+  UseMultipleSelectionReturnValue as UseMultiSelectPrimitiveReturnValue,
+  UseMultipleSelectionStateChange as UseMultiSelectPrimitiveStateChange,
```

Comment:
Are these all already exported from the dropdown library package from Downshift? Did we do that already?

Reply:
Yeah, the dropdown library exports [_all_ of downshift](https://github.com/twilio-labs/paste/blob/main/packages/paste-libraries/dropdown/src/index.tsx#L6).

Reply:
Oh sweet, that makes things super easy. Dunno why I thought we only exported the hooks